src/DependencyInjection/DbToolsExtension.php

```diff
@@ -23,7 +26,10 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new YamlFileLoader($container, new FileLocator(\dirname(__DIR__).'/../config'));
         $loader->load('services.yaml');
 
-        $container->setParameter('db_tools.storage_directory', $config['storage_directory']);
+        if (isset($config['storage_directory'])) {
+            \trigger_deprecation('makinacorpus/db-tools-bundle', '1.0.1', '"db_tools.storage_directory" configuration option is deprecated and renamed "db_tools.storage.root_dir"');
+        }
```

Comment:
I don't know if this is necessary as you set this configuration as deprecated in `DbToolsConfiguration.php`.

But I guess we can keep it